a3p-integration/.yarn/patches/@agoric-synthetic-chain-npm-0.1.1-d26a740f05.patch

```diff
@@ -0,0 +1,126 @@
+diff --git a/dist/cli/cli.js b/dist/cli/cli.js
+index c59a54224c101648c5507f451bfe0b2cb7f420a5..24ef27a882805df4a969a96ab28dad15b8ee5479 100755
+--- a/dist/cli/cli.js
++++ b/dist/cli/cli.js
+@@ -14,7 +14,9 @@ import path from "node:path";
+ var defaultConfig = {
+   // Tag of the agoric-3 image containing all passed proposals
+   // Must match the Bake file and CI config
+-  fromTag: "latest"
++  fromTag: "latest",
++  repositoryColon: "ghcr.io/agoric/agoric-3-proposals:",
++  sdkRepositoryColon: "ghcr.io/agoric/agoric-sdk:",
+ };
+ function readBuildConfig(root2) {
+   const packageJsonPath = path.join(root2, "package.json");
+@@ -25,7 +27,8 @@ function readBuildConfig(root2) {
+ }
+ var buildProposalSubmissions = (proposals2) => {
+   for (const proposal of proposals2) {
+-    if (!("source" in proposal && proposal.source === "build")) continue;
++    if (!("source" in proposal && proposal.source === "build"))
++      continue;
+     console.log(
+       "Refreshing submission for",
+       proposal.proposalIdentifier,
+@@ -63,7 +66,6 @@ import fs3 from "node:fs";
+ import assert from "node:assert";
+ import fs2 from "node:fs";
+ import * as path2 from "node:path";
+-var repository = "ghcr.io/agoric/agoric-3-proposals";
+ function readInfo(proposalPath) {
+   assert(
+     proposalPath === proposalPath.toLowerCase(),
+@@ -105,7 +107,7 @@ function readProposals(proposalsParent) {
+ function imageNameForProposal(proposal, stage2) {
+   const target = `${stage2}-${proposal.proposalName}`;
+   return {
+-    name: `${repository}:${target}`,
++    name: `${buildConfig.repositoryColon}${target}`,
+     target
+   };
+ }
+@@ -143,7 +145,7 @@ RUN /usr/src/upgrade-test-scripts/run_prepare_zero.sh
+   RESUME(fromTag) {
+     return `
+ ## RESUME
+-FROM ghcr.io/agoric/agoric-3-proposals:${fromTag} as use-${fromTag}
```

Comment:
why are these Docker vars necessary? the Dockerfile is generated by the CI job. Couldn't it just produce the desired file contents?

Reply:
yes, very much

Reply:
I've removed any dependencies on a synthetic-chain patch, and reduced the postprocessing of the `prepare-build`-generated files to a single `sed`.  Let's talk about the best way to make that change robust.